Translations/fr.po

```diff
@@ -1060,7 +1060,7 @@ msgstr[1] "{:s} pièces d'or"
 
 #: Source/control.cpp:863
 msgid "Requirements not met"
-msgstr "Requis insuffisant"
+msgstr "Requis non satisfaits"
```

Comment:
That still does not sound very familiar to me.
Is a bit far from the original wording but may be something like
"Vous n'avez pas les conditions requises"

Reply:
Totally agree. I was lazy and tried to improve while keeping a short text, not knowing where it was displayed.

Reply:
I tried in game, and it tightly fits in the frame.

```suggestion
msgstr "Vous n'avez pas les conditions requises"
```